File: ruby/6.1.bonobus.rb
class Bonobus
    def initialize
        @viajes = 10
    end

    def usar
        @viajes = @viajes - 1
    end

    def info
        puts "viajes disponibles: #{@viajes}"
    end
end

mi_bonobus = Bonobus.new
mi_bonobus.info
mi_bonobus.usar
mi_bonobus.info

File: python/6.1.bonobus.py
class Bonobus:
    def __init__(self):
        self.__viajes = 10

    def usar(self):
        self.__viajes = self.__viajes - 1

    def info(self):
        print(f"viajes disponibles: {self.__viajes}")

mi_bonobus = Bonobus()
mi_bonobus.info()
mi_bonobus.usar()
mi_bonobus.info()
